### PR TITLE
[v1.x] ONNX Supoort for MXNet repeat op

### PR DESCRIPTION
**Pull request by Zha0q1**

### CODE REVIEW

**Review by josephevans — APPROVED** [review]
LGTM!

File: tests/python-pytest/onnx/test_operators.py [hunk 1]
```diff
@@ -351,3 +351,12 @@ def test_onnx_export_softmax(tmp_path, dtype):
     M4 = def_model('softmax', use_length=True, axis=1)
     l4 = mx.nd.array([[2,0,3,1],[0,1,0,0]], dtype=int)
     op_export_test('softmax_4', M4, [x, l4], tmp_path)
+
+
+@pytest.mark.parametrize('dtype', ['float16', 'float32', 'float64', 'int32', 'int64'])
+@pytest.mark.parametrize('axis', [None, 0, 1, 2])
```
Comment (waytrue17) [inline tests/python-pytest/onnx/test_operators.py:357]: Shall we also test negative axis like -1 or -2?

Comment (Zha0q1) [reply tests/python-pytest/onnx/test_operators.py]: ohh good point!

**Review by waytrue17 — APPROVED** [review]
LGTM, thanks

**Pull request merged**